routes/api.js

```diff
@@ -0,0 +1,67 @@
+var express = require('express');
+var router = express.Router();
+
+
+// log request
+router.use(function(req, res, next) {
+    console.log(req.method, req.url);
+    next(); 
+});
+
+//TODO: Check if a user is permitted to use station 
+router.route('/user-access')
+.post(function(err, req, res, next) {
```

Comment:
out of curiosity is there any reason that you did `.route` call first to set the route and then `.post`? As opposed to just doing the `.post` call which can include the route and callback function?

Reply:
@danieleynis just following documentation  __*http://expressjs.com/en/api.html#router.route*__

>the following code shows how to use router.route() to specify various HTTP method handlers.

```
router.route('/users/:user_id')
.get(function(req, res, next) {
  res.json(req.user);
})
.put(function(req, res, next) {
  // just an example of maybe updating the user
  req.user.name = req.params.name;
  // save user ... etc
  res.json(req.user);
})
.post(function(req, res, next) {
  next(new Error('not implemented'));
})
.delete(function(req, res, next) {
  next(new Error('not implemented'));
});```

Reply:
@danieleynis since I'm not currently chaining I'll change it to ```router.post('/route', function ...)

Reply:
@jstar1 I think it's fine for now if it works, but up to you.